signer/tests/integration/block_observer.rs

```diff
@@ -1415,3 +1420,251 @@ async fn block_observer_updates_dkg_shares_after_observing_bitcoin_block() {
 
     testing::storage::drop_db(db).await;
 }
+
+#[test_log::test(tokio::test)]
+async fn block_observer_ignores_coinbase() {
+    let mut rng = rand::rngs::StdRng::seed_from_u64(51);
```

Comment:
We now can use `signer::testing::get_rng()` here!

Reply:
Nice! Changed in [77a428b](https://github.com/stacks-sbtc/sbtc/pull/1592/commits/77a428bb29c1e1e90b12708b1b9ff27cfe890183)